eng/versioning/version_client.txt

```diff
@@ -44,6 +44,6 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: the unrelease dependencies do not require the empty line as their versions
 # will not be manipulated with the automatic PR creation code.
-unreleased_com.azure:azure-core;1.3.0-beta.2
+unreleased_com.azure:azure-core;1.3.0
 unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
 unreleased_com.azure:azure-core-http-netty;1.4.0-beta.2
```

Comment:
azure-core-amqp and azure-core-http-netty also need to be updated

Reply:
Changed